host_reference/dslash_reference.cpp

```diff
@@ -227,7 +227,7 @@ void verifyWilsonTypeInversion(void *spinorOut, void **spinorOutMulti, void *spi
       double src2 = norm_2(spinorIn, Vh * spinor_site_size, inv_param.cpu_prec);
       double l2r = sqrt(nrm2 / src2);
 
-      printfQuda("Shift %d residuals: (L2 relative) tol %g, QUDA = %g, host = %g; (heavy-quark) tol %g, QUDA = %g\n", i,
+      printfQuda("Shift %2d residuals: (L2 relative) tol %9.6e, QUDA = %9.6e, host = %9.6e; (heavy-quark) tol %9.6e, QUDA = %9.6e\n", i,
```

Comment:
I'm curious what the motivation for these formatting changes is?

Reply:
So that the print out numbers are aligned, which looks nicer (at least in my mind).